GoogleSignInSwift/Sources/GIDSwiftUISignInButton.swift

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+import CoreGraphics
+
+// MARK: - Sign-In Button
+
+/// A Google Sign In button to be used in SwiftUI.
+@available(iOS 13.0, macOS 10.15, *)
+public struct GIDSwiftUISignInButton: View {
```

Comment:
Do we want the `GID` prefix in the Swift environment?

Reply:
What do you think about renaming to `GoogleSignInButton`

Reply:
Yes.  I think either `GoogleSignInButton` or `SignInButton`.

With namespacing:

```
GoogleSignInSwift.GoogleSignInButton
or
GoogleSignInSwift.SignInButton
```

and eventually:

```
GoogleSignIn.GoogleSignInButton
or
GoogleSignIn.SignInButton
```

Reply:
I went with `GoogleSignInButton`.